class6/DistinctElements.java

```diff
@@ -7,14 +10,37 @@
  */
 public class DistinctElements {
     private static boolean isDistinct(int[] nums){
-
+    	if(nums==null){
+    		return true;
+    	}
+    	else if(nums.length==1){
+    		return true;
+    	}
+  
+    	Set<Integer> mySet = new HashSet<>();
```

Comment:
very good code & efficient.. one small suggestion or rather improvement in writing neat code..
instead of this, for(int i=0;i<nums.length;i++)

try writing like this,  for(int i = 0; i < nums.length; i++)
always space between operators and operands, gives more clarity and readability in the code.. I learnt this tip recently... similarly you can use for if condition also..